README.md

```diff
@@ -10,7 +10,7 @@ A `VintageNet` technology for using mobile connections.
 ```elixir
 def deps do
   [
-    {:vintage_net_mobile, "~> 0.1.0"}
+    {:vintage_net_mobile, "~> 0.1"}
```

Comment:
I have mixed feelings here. For pre-1.0.0 releases, updates to the "minor" numbers are potentially breaking changes. I've been keeping all three digits to avoid breaking changes. The downside is that you have to be vigilant since it's easy to miss updates if `mix hex.outdated` isn't in your workflow.

Reply:
I updated!